presto-server/src/main/provisio/presto.xml

```diff
@@ -1,6 +1,6 @@
 <runtime>
     <!-- Target -->
-    <archive name="${project.artifactId}-${project.version}.tar.gz" />
+    <archive name="${project.artifactId}-${project.version}.tar.gz" hardLinkIncludes="**/*.jar" />
```

Comment:
How does the produced .tar.gz work on a file system that doesn't support hardlinks?
How does it work on Windows?

Reply:
I've literally used Windows five times in my entire life, so someone would need to try it but I assume they would use the `--hard-dereference` option or equivalent. The tar implementations on Windows might even dereference hardlinks given they aren't supported.